app/components/home/transaction-list/transaction-list-empty.tsx

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
-import { Text, Flex } from '@stacks/ui';
+import { Text, Flex, Box, color } from '@stacks/ui';
 
+import { EmptyTxList } from '@components/icons/empty-tx-list';
 import { useNavigatorOnline } from '@hooks/use-navigator-online';
 import { templateTxBoxProps } from './transaction-list-item-pseudo';
 
 export const TransactionListEmpty = () => {
   const { isOnline } = useNavigatorOnline();
   return (
     <Flex {...templateTxBoxProps}>
-      <Text textStyle="body.small" display="block" textAlign="center" mb="tight">
+      <Box maxWidth="180px" mb="loose">
+        <EmptyTxList />
+      </Box>
```

Comment:
could be this
```suggestion
      <Box as={EmptyTxList} maxWidth="180px" mb="loose" />
```

Reply:
noted but don't want to wait for ci to rebuild